package.json

```diff
@@ -36,11 +36,10 @@
     "glob": "^7.1.2",
     "humanize": "0.0.9",
     "ioslib": "^1.7.2",
-    "lodash.defaultsdeep": "4.6.0",
```

Comment:
@sgtcoolguy `lodash.defaultsdeep` is used in [titanium_mobile_windows/cli/commands/_build/generate.js#L11](https://github.com/appcelerator/titanium_mobile_windows/blob/master/cli/commands/_build/generate.js#L11) and removing it is causing build error now. Can we revive this or is there anything titanium_mobile_windows can handle by itself?